test/runtests.jl

```diff
@@ -2,13 +2,25 @@ using Test, Random, LinearAlgebra, Statistics
 using Lighthouse
 using Lighthouse: plot_reliability_calibration_curves, plot_prg_curves, plot_pr_curves,
                   plot_roc_curves, plot_kappas, plot_confusion_matrix,
-                  evaluation_metrics_plot, plot_combined
+                  evaluation_metrics_plot
 using Base.Threads
+using CairoMakie
 
-# Set up plotting backend for Plots.jl (GR)
-using Plots
-gr()
-GR.inline("png")
+# Needs to be set for figures
+# returning true for showable("image/png", obj)
+# which TensorBoardLogger.jl uses to determine output
+CairoMakie.activate!(type="png")
+plot_results = joinpath(@__DIR__, "plot_results")
+isdir(plot_results) || mkdir(plot_results)
```

Comment:
```suggestion
mkpath(plot_results)
```

Reply:
I actually just added `isdir(plot_results) ? rm(plot_results; force=true, recursive=true) : mkdir(plot_results)`, to make sure we don't have any old pictures in there ...

Reply:
I just added this:
```julia
# Remove any old plots
isdir(plot_results) && rm(plot_results; force=true, recursive=true)
mkdir(plot_results)
```
To make sure we don't work with old plots...

Reply:
Nice!